libindy/Cargo.toml

```diff
@@ -66,7 +66,13 @@ lazy_static = "1.0"
 named_type = "0.1.3"
 named_type_derive = "0.1.3"
 byteorder = "1.0.0"
+log-panics = "2.0.0"
 [dependencies.uuid]
 version = "0.5.0"
 default-features = false
-features = ["v4"]
\ No newline at end of file
+features = ["v4"]
+
+[target.'cfg(target_os = "android")'.dependencies]
+android_logger = "0.5"
+[target.'cfg(any(target_os = "android", target_os = "ios"))'.dependencies]
+rusqlite = { version = "0.13.0", features=["bundled"] }
```

Comment:
please specify common and android rusqlite dependence as neighbors to avoid partial update in the future